testdata/book/http_multipart.yml

```
@@ -16,3 +16,14 @@ steps:
               upload1: ../dummy.jpg
     test: |
       current.res.status == 201
+  fileupload2:
+    desc: Post /upload with array
+    req:
+      /upload:
+        post:
+          body:
+            multipart/form-data:
+              - file: ../dummy.png
+              - file: ../dummy.jpg
```

I think it is good to support multiple files in the following way.
The reason is that the format of inputs other than files remains the same.

```suggestion
              file: 
                - ../dummy.png
                - ../dummy.jpg
```